### PR TITLE
Cc-123 methods used in the scientific study of earth

### PR DESCRIPTION
**Pull request by Birce8**

### CODE REVIEW

**Comment by nickurak** [comment]
TLDR -- you should use the _"squash and merge"_ option when merging this PR in (or do something else to edit the history of these commits).

Details:

A quick reminder, since I saw this PR was removing a bunch of files which, correctly, shouldn't be in this repository:

- Because the commits that *add* those files are still in this PR, those commits would normally be added to the repository, and effectively those large files will still be in the repo **forever**.
- You can correct that by squashing those commits together (so they effectively cancel out).
- GitHub can do that automatically for you by selecting the "Squash and Merge" option
- You can also do the squash yourself on your machine, and use the "--force" option to push a corrected version of these commits to the PR **before** the PR gets merged.

I'm happy to help out with that if anyone would like me to walk though the steps needed there or if anyone has any questions -- just ping me on slack.
